pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.de-de.md

```diff
@@ -38,79 +43,41 @@ details[open]>summary::before {
+| **Step 1**  | ❌           | **Multi-vDC**                            | **Migrate data to a single datacentre.**                   | - Can only be migrated if it has only one data centre in a vSphere managed on OVHcloud. If not, make sure to migrate your data in the datacenter that will be migrated.                                                | [Migrating an infrastructure to a new vDC](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/service-migration-vdc)                                            |                                                                                                                                                                                                                                                                                                                                                  
+| **Step 2**  | ❎           | **FT (fault tolerance)**                 | **Disable FT from VMs in managed vSphere on OVHcloud.**    |                                                                                                                                                                                                                        | [VMware Fault Tolerance](/pages/bare_metal_cloud/managed_bare_metal/vmware_fault_tolerance)                                                                                     |                                                                                                                                                                                                                                                                                                                                                            
+| **Step 3**  | ️❎          | **DRS affinity/anti-affinity**           | **Reconstitution of affinity/anti-affinity rules in VCD.** | - Between VMs migration can be migrated to VCD, but between VMs and hosts cannot be migrated.                                                                                                                          | [VMware DRS distributed resource scheduler](/pages/bare_metal_cloud/managed_bare_metal/vmware_drs_distributed_resource_scheduler)                                               |                                                                                                                                                                                                                                                                                                                                               
```

Comment:
"Between VMs migration can be migrated to VCD, but between VMs and hosts cannot be migrated." I really don't understand that sentence. It doesn't make sense.